tests/unit/test_atomic_json.py

```
@@ -0,0 +1,212 @@
+
+
+def test_append_ndjson_concurrent_no_torn_lines(tmp_path):
+	path = str(tmp_path / 'results.ndjson')
+	ctx = mp.get_context('fork')
+	procs = [ctx.Process(target=_ndjson_proc_worker, args=(path, w, 50)) for w in range(4)]
+	for p in procs:
+		p.start()
+	for p in procs:
+		p.join(60)
+		assert p.exitcode == 0
+	lines = [l for l in open(path).read().splitlines() if l]
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)